src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,20 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_in_process(extra_env):
+  """Decide whether to use Radamsa in process. If yes, add the path to the
+  radamsa so to LD_PRELOAD in |extra_env| and return True."""
```

Comment:
I would write `shared object` here. `so` is confusing since it is an english word.